### DISCUSSION TITLE
Shipping methods, categories and zones should be config, not entries

### DISCUSSION THREAD

**Discussion by venturini-dev** [craftcms/commerce#3205] [Ideas]
Shipping methods, categories and zones are treated like entries. However, compared to normal entries, they cannot be exported and imported from the backend. Because they are entries, they cannot be synced between environments using config files.
The only option is to migrate them manually as content.
In our experiences shipping settings rarely change, especially between environments. Usually we want them to be synced, so that we can test shipping in different environments. I cannot think of a use case where entries are an advantage for shipping settings..
Can you consider changing these to config files, or at least make them easily exportable like standard entries, and make them importable via feed me?
What is the recommended practice to migrate shipping setups between enviroments?
thanks!

**Comment by lukeholder** [comment]
Hi @venturini-dev thanks for the feedback.

You are correct at the moment we treat shipping rules/zones etc as content on the site since we expect store administrators could be modifying shipping rules or categories as prices for shipping could change, and they would not need to get the developer involved.

Unfortunately we can't have it both ways (project config AND in modifiable in production) at the moment, so the way it is at the moment is likely to remain in the near term. I will keep this discussion open to see if anyone else has the same opinion.

Maybe in the future certain things like prices could we exposed to the store administrator without allowing modification of all the rules/zones.

**Reply by venturini-dev** [reply]
hi @lukeholder,
thanks for the info, now that you explain this to me I see why you opted to treat them as content.

Do you think it would be possible to add an export method so we can move these between environments easily, for instance exporting from staging end importing in production using something like feedme?
I guess that would be a full.. "entrification"?

Thanks again, keep up the great work!
